src/components/Lounge.js

```diff
@@ -1,12 +1,86 @@
-import React from 'react';
+import React, { useState,useEffect } from 'react';
+import "./Lounge.css";
+
+var timer;
+var count;
+function Lounge({curr}){
+    const [second,setsecond]=useState(0);
+    const [minute,setminute]=useState(0);
+    const [countm,setcountm]=useState("");
+    const [counts,setcounts]=useState("");
+    useEffect(()=>{
+        timer= setInterval(()=>{
+            setsecond(second+1);
+            if(minute<9){
+                setcountm("0");
+            }
+            else{
+                setcountm("");
+            }
+            if(second<9){
+                setcounts("0");
+            }
+            else{
+                setcounts("");
+            }
+        if(second===59){
+            setminute(minute+1);
+           
+            
+            setsecond(0);
+        }},1000)
+     return()=>clearInterval(timer);   
+    }
+    );
+
 
 
-function Lounge(){
     return(
         <div>
-            <h1>Hello There,</h1> <h1>Wait till others arive....</h1>
-            <h1>Quizz will start shortly</h1>
-        </div>
+              <style
+    dangerouslySetInnerHTML={{
+      __html:
+        "\n\t\t\th1 {\n\t\t\t\tcolor:white;\n\t\t\t}\n\t\t\tdiv.gfg {\n\t\t\t\tmargin:5px;\n\t\t\t\tpadding:5px;\n\t\t\t\tcolor: white;\n\t\t\t\twidth: 250px;\n\t\t\t\theight: 100%;\n\t\t\t\toverflow: auto;\n\t\t\t\ttext-align:justify;\n        position:fixed;\n        \n\n\t\t\t}\n\t\t"
+    }}
+  />
+    <div className="gfg scroll">
+        <h1 className='msg'>MESSAGE</h1>
+
+
+        <hr></hr><hr></hr>
```

Comment:
Please State the use of these hr tags and remove if redundent

Reply:
these <hr> tag are the horizontal rules under message text for decoration purposes